prometheus/manifests/prometheus/010-replication-pvc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```

Comment:
This is not a PVC, it is a StorageClass. The file is called 010 pvc. 

Please change this. It will bring confusion to a user if the SC already exist and fails. I guess that changing the name of the File is enough